samples/msal-node-samples/auth-code-distributed-cache/README.md

```diff
@@ -0,0 +1,82 @@
+# MSAL Node Standalone Sample: Web app using Authorization Code Flow
+
+This sample demonstrates how to implement an MSAL Node [confidential client application](../../../lib/msal-node/docs/initialize-confidential-client-application.md) calling (1) Microsoft Graph directly using the [OAuth 2.0 Authorization code grant](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) and (2) a protected web API (aka *middle-tier*) which in turn calls Microsoft Graph using the [OAuth 2.0 on-behalf-of flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) (see also the sample: [on-behalf-of-distributed-cache](../on-behalf-of-distributed-cache)).
+
+In addition, this sample uses the MSAL Node [DistributedCachePlugin](../../../lib/msal-node/src/cache/distributed/DistributedCachePlugin.ts) to implement the [distributed token caching](../../../lib/msal-node/docs/caching.md#performance-and-security) pattern. Here, the cache is persisted via [Redis](https://redis.io/) and [node-redis](https://github.com/NodeRedis/node-redis).
```

Comment:
Future: Once your doc PR is updated, link here for reference please to set the context (if not already done, I couldn't tell).